third_party/move/move-vm/types/src/code/cache/types.rs

```diff
@@ -61,6 +67,11 @@ where
         Self::Verified(Arc::new(verified_code))
     }
 
+    /// Returns new verified code from [Arc]ed instance.
+    pub fn from_arced_verified(verified_code: Arc<V>) -> Self {
```

Comment:
nit: `from_arc_to_verified`

Reply:
It's actually not `to`, it is already verified we just wrap it in module code struct? How about `from_verified_arced` or `from_verified_arc`?

Reply:
I meant "from `arc to verified` to this", but realized it has double meaning :)